### ISSUE TITLE
Code from GitHub is different from code imported by Maven

### ISSUE THREAD

**Issue by matt12f** [AtlisInc/Nominatim-API#7]
Hi, I was having some problems with the use of your library (having imported it in my project using the Maven dependency) so I downloaded the source code to look into it a little deeper and I noticed that the code imported by Maven is different from the one here on GitHub. 

The code from GitHub is probably more recent, because it works just fine and doesn't use HttpsUtils, that returned a 403 error (in the code that's imported by Maven).

If you know why there would be such a weird discrepancy let me know, it's probably just a matter of updating the code that Maven has (I don't really know how it works)

In the meanwhile I'll manually add the source code of your library from the working version here on GitHub.

Anyway, thanks for a simple and functional library :)

**Comment by michaelassraf** [comment]
Hi @matt12f,

I actually lost my maven certificates..... 
If you know how to upload artifacts to maven I will be happy to transfer this repo ownership.

Michael

**Comment by matt12f** [comment]
Hi @michaelassraf,

No problem, I've solved anyway by now. I was able to import the Library in my project and it works great. I don't remember how I did it to be honest... but I probably downloaded the source code and compiling it locally avoiding the import through Maven.

I would be happy to help but I don't know enough about how maven works to take ownership of this repo. It's still great that it's out there though, even if it's not on maven.

Matteo
